packages/mason/lib/src/hooks.dart

```diff
@@ -438,7 +438,7 @@ abstract class HookContext {
 }
 
 final _runRegExp = RegExp(
-  r'((void||Future<void>)\srun\(HookContext)',
+  r'((void||Future<void>)\srun\((\s.*)?HookContext)',
```

Comment:
```suggestion
  r'((void||Future<void>)\srun\((\s*)?HookContext)',
```

Why is the `.` necessary? I don't think we want to allow any characters here.